server/cluster/cluster.go

```diff
@@ -1302,14 +1299,21 @@ func (c *RaftCluster) resetMetrics() {
 	c.resetHealthStatus()
 }
 
-func (c *RaftCluster) collectClusterMetrics() {
-	c.RLock()
+func (c *RaftCluster) collectStatsMetrics() {
+	c.Lock()
+	defer c.Unlock()
 	if c.regionStats == nil {
-		c.RUnlock()
 		return
 	}
 	c.regionStats.Collect()
 	c.labelLevelStats.Collect()
+	c.lastRegionStats = c.regionStats.Clone()
+	c.labelLevelStats = statistics.NewLabelStatistics()
```

Comment:
Do we need to handle `labelLevelStats` the same way as `regionStats`?

Reply:
No, we don't, `lastRegionStats` is used for API.